tests/vmi_configuration_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	"time"
 	"unicode"
 
+	nodev1 "k8s.io/api/node/v1"
```

Comment:
nit: wrong import order isn't it ? Please put it back where the `nodev1beta` was